cordformation/src/main/kotlin/net/corda/plugins/Dockerform.kt

```diff
@@ -100,7 +100,7 @@ open class Dockerform @Inject constructor(objects: ObjectFactory) : Baseform(obj
                             "$nodeBuildDir/drivers:/opt/corda/drivers"
                     ),
                     "ports" to listOf(it.rpcPort.get(), it.config.getInt("sshd.port")),
-                    "image" to (dockerImage ?: "corda/corda-zulu-${it.runtimeVersion().toLowerCase()}")
+                    "image" to (dockerImage ?: "corda/corda-zulu-java1.8-${it.runtimeVersion().toLowerCase()}")
```

Comment:
This kind of thing shows why it is better to _force_ the user to specify the image they want to use in the first place. If some people had their way, the plugins would be responsible for :sparkles: **automagically** :sparkles: configuring everything from CorDapp developers' compiler settings onwards. And consequently, every subsequent misconfiguration problem would become the plugins' problem too.

Reply:
The users can now specify the Docker image by using dockerImage property, but is provided as optional. Should we make this change this?

Reply:
I think that's a breaking change so you probably can't - unless we have a different policy around the plugins.

Reply:
To be blunt: _Yes_, I would prefer that the image were a mandatory `@Input` property without a default. Because otherwise we'll be right back here when (say) we switch to Java 11 or someone decides that the default should be different for any other reason.

Reply:
> I think that's a breaking change so you probably can't - unless we have a different policy around the plugins.

It was broken back when the name of the base image changed. And we really don't want to open the Pandora's Box of trying to hard-code different default values for different versions of Corda into the plugins. So maybe it's better to leave it broken for now and help people migrate? And _then_ we kill the default entirely.

Reply:
Come to think of it, this PR is _itself_ a breaking change for those using Corda 4.3 and earlier.

Reply:
Made the dockerImage property mandatory and also update the docs to reflect this change